packages/components/src/products.tsx

```diff
@@ -270,12 +270,14 @@ export const PRODUCTS_MENU_LIST: MenuItem['items'] = Object.fromEntries(
         newWindow: true,
         title: (
           <div className="flex items-center gap-2">
-            <Logo
+            <div
               className={cn(
                 'size-4 translate-y-[0.25px]',
-                i > 3 && 'rounded-sm bg-gray-500 text-[8px] text-white dark:bg-white/10',
+                i > 3 && 'rounded-sm bg-gray-500 text-[8px] text-white dark:bg-white/10 p-0.5',
```

Comment:
```suggestion
                i > 3 && 'rounded-sm bg-gray-500 text-white dark:bg-white/10 p-0.5',
```
not text inside

Reply:
Oh yeah, this should be lower there on the logo, good catch.

Reply:
We want `text-[8px]` to override the `text-sm`.

<img width="523" alt="image" src="https://github.com/user-attachments/assets/2c752c11-81be-4ae4-9c8b-75fa671c7656">

Reply:
ah yeah I forgot that we have text (not SVG) logos